mod/wikirate/set/type_plus_right/project/project_task_progress.rb

```diff
@@ -0,0 +1,15 @@
+def virtual?; true end
+format :html do
+  view :raw do |args|
+    
+    closed_task_number = Card.search :found_by=>card.cardname.left+"+tasks",:right_plus=>["Status",{:refer_to=>"Closed"}],:return=>"count"
+    all_task_number = Card.search :found_by=>card.cardname.left+"+tasks",:return=>"count"
+
+    if all_task_number == 0 
+      0
+    else
+      (closed_task_number.to_i*100)/all_task_number.to_i
+    end
+  end
```

Comment:
We can have `project+progress` as a number. `project+progress+progress bar` can show the progress bar based on the number users input in `project+progress` card.

I can remove this code if the above solution is fine to you.